### PR TITLE
doc: initial doc for comptime resources

### PR DESCRIPTION
**Pull request by felipensp**
Initial doc for comptime resources

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -5591,6 +5591,10 @@ print($embed_file(@FILE).to_string())
 Having built-in JSON support is nice, but V also allows you to create efficient
 serializers for any data format. V has compile time `if` and `for` constructs:
 
+#### .fields
+
+You can iterator over struct fields using `.fields`, it also works with generic types (e.g. `T.fields`) and generic arguments (e.g. `param.fields` where `fn gen[T](param T) {`).
```
Comment (JalonSolov) [inline doc/docs.md:5596]:
```suggestion
You can iteratr over struct fields using `.fields`, it also works with generic types (e.g. `T.fields`) and generic arguments (e.g. `param.fields` where `fn gen[T](param T) {`).
```

File: doc/docs.md [hunk 2]
```diff
@@ -5609,6 +5613,112 @@ fn main() {
 // name is of type string
 ```
 
+#### .values
+
+You can read [Enum](#enums) values and its attributes.
```
Comment (JalonSolov) [inline doc/docs.md:5618]:
```suggestion
You can read [Enum](#enums) values and their attributes.
```

**Comment by felipensp** [comment]
Thanks @JalonSolov Feel free to edit it.

**Review by spytheman — APPROVED** [review]
Excellent work.

**Comment by JalonSolov** [comment]
Looks good to me.  The only problems I saw were the ones I mentioned, and you fixed.  :-)

**Review by spytheman — APPROVED** [review]

**Pull request merged**